src/concrete/ml/sklearn/linear_model.py

```diff
@@ -398,6 +400,8 @@ def _fit_encrypted(
                 optimization. Default to None.
             intercept_init (Optional[numpy.ndarray]): The initial intercept to warm-start the
                 optimization. Default to None.
+            is_partial_fit (bool): Indicates if this fit represents a partial fit. A partial fit is
```

Comment:
basically I removed `_encrypted_fit_one_step` (made for `partial_fit`) and added this new param in `_fit_encrypted` to handle the few differences with between `fit` and `partial_fit`

Reply:
That's indeed probably a better solution than the 2 method one we had before!